cves/CVE-2013-0305.yml

```diff
@@ -95,10 +108,16 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: true
+
+  code_answer: |
+    It is clear that initially there was a unit test (tests/regressiontests/admin_views/tests.py), however it 
+    didn't properly test for restricting access.
```

Comment:
Maybe be more clear as to what the unit test originally focused on so that we have a better understanding what it did improperly.

Reply:
I should've been much more clear initially, there was a file present for unit testing the admin interface, however no function to check permissions for history view in particular was present before fixing the vulnerability.